en/components/combo.md

```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
-`line`
 
 <img class="responsive-img" src="../images/combo_border.png" srcset="../images/combo_border@2x.png 2x" />
-`border`
 
 <img class="responsive-img" src="../images/combo_box.png" srcset="../images/combo_box@2x.png 2x" />
-`box`
 
-## Interaction State
+## Enabled and Disabled States
 
-Each of the three Combo input types can be inserted in an enabled or disabled state.
+Each of the two Combo input types can be inserted in an enabled or disabled state. In Figma the input can be disabled from the property panel.
 
 <img class="responsive-img" src="../images/combo_enabledstate.png" srcset="../images/combo_enabledstate@2x.png 2x" />
 <img class="responsive-img" src="../images/combo_disabledstate.png" srcset="../images/combo_disabledstate@2x.png 2x" />
 
-## State
+## States
 
-When the user interacts with the Combo, its input goes through various selection states: **idle** with a placeholder in the place of the content, focused when the dropdown is open, filled once the user has finished with selection and moved on. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
+When the user interacts with the Combo, its input goes through various selection states: **Idle** with a placeholder in the place of the content, **Focused** when the drop down is open, **Filled** once the user has finished with selection and moved on. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
 
 <img class="responsive-img" src="../images/combo_idle.png" srcset="../images/combo_idle@2x.png 2x" />
-`idle`
 
 <img class="responsive-img" src="../images/combo_focused.png" srcset="../images/combo_focused@2x.png 2x" />
-`focused`
 
 <img class="responsive-img" src="../images/combo_filled.png" srcset="../images/combo_filled@2x.png 2x" />
-`filled`
 
 Every experienced designer uses constraints wisely to limit the user input and avoid invalid states, hence the availability of validation styles. Through the available validation styles, the Combo is consistent with the standard [Input](input.md) and equipped for sophisticated designs that display success, warning, and error visuals.
 
 <img class="responsive-img" src="../images/combo_success.png" srcset="../images/combo_success@2x.png 2x" />
 <img class="responsive-img" src="../images/combo_warning.png" srcset="../images/combo_warning@2x.png 2x" />
 <img class="responsive-img" src="../images/combo_error.png" srcset="../images/combo_error@2x.png 2x" />
 
-In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Adobe XD we are using the `Component States` paradigm to let you easily switch between them.
+In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Figma and Adobe XD we are using the `Component States` paradigm to let you easily switch between them.
```

Comment:
I'm not sure whether its appropriate to say that in Figma we use the "Component States" paradigm like in Adobe XD. I think it's better to say we have a property for this, which can be found on the properties panel or something similar.